src/main/java/com/CezaryZal/manager/LoginService.java

```diff
@@ -0,0 +1,64 @@
+package com.CezaryZal.manager;
+
+import com.CezaryZal.entity.User;
+import com.CezaryZal.entity.UserLogin;
+import com.CezaryZal.exceptions.IncorrectInput;
+import com.CezaryZal.manager.db.service.UserServiceImp;
+import com.CezaryZal.manager.builder.TokenBuilder;
+import com.CezaryZal.manager.filters.comparator.PasswordComparator;
+import com.CezaryZal.manager.filters.validator.UserLoginValidatorService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import javax.security.auth.login.AccountNotFoundException;
+
+
+@Service
+public class LoginService {
+
+    private UserServiceImp userServiceImp;
+    private TokenBuilder tokenBuilder;
+    private UserLoginValidatorService userLoginValidator;
+    private PasswordComparator passwordComparator;
+
+    public LoginService(UserServiceImp userServiceImp, TokenBuilder tokenBuilder,
+                        UserLoginValidatorService userLoginValidator, PasswordComparator passwordComparator) {
+        this.userServiceImp = userServiceImp;
+        this.tokenBuilder = tokenBuilder;
+        this.userLoginValidator = userLoginValidator;
+        this.passwordComparator = passwordComparator;
+    }
+
+    public ResponseEntity<String> getTokenByUserLogin(UserLogin inputUserLogin) {
```

Comment:
To nie powinno zwracać ResponseEntity a nieowrappowany  obiekt. ResponseEntity powinno się pojawić dopiero w warstwie controllera.

Reply:
poprawione